Examples/JavaScript/JavaScript Essentials/Functions.ts

```diff
@@ -0,0 +1,63 @@
+//// { order: 2 }
```

Comment:
I would turn off noImplicitAny for this example and remove the ts-ignores, since the examples don't rely on either one really.